### PR TITLE
Janky

### PR DESCRIPTION
**Pull request by GoatEXE**
# Hmm.cs
Hmm.

### CODE REVIEW

File: Scripts/Characters/Player/Controls.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
+using Godot;
+using System;
+
+public partial class Controls : CharacterBody2D
+{
+    private Player _player;
```
Comment (LarsenEndeavors) [inline Scripts/Characters/Player/Controls.cs:6]: Sort of an IoC (Inversion of Control) approach here, totally a valid approach, what did you find to be the biggest stumbling block?

Comment (GoatEXE) [reply Scripts/Characters/Player/Controls.cs:6]: @LarsenEndeavors I'm super unfamiliar with programing styles. If you have any good reading material for that, I'd be down to get into that.

Simply put, it's me. I have a very pythonic way of thinking about programing. I can indefinitely import classes into python and control them all easily. From what I can tell with C#, I can extend one class (`CharacterBody2D : MainCharacter`). It puts a damper on my imagination somehow. As a result, I'm pretty sure I'm inventing janky ways of navigating these scripts/ideas.

Comment (LarsenEndeavors) [reply Scripts/Characters/Player/Controls.cs:6]: No worries, we should have a meeting about this, I'll get something together and then you can ask questions as we go, hopefully it'll be better than random text

Comment (LarsenEndeavors) [reply Scripts/Characters/Player/Controls.cs:6]: As for books to read, boy I have books for days haha, I'll add some to our Drive, but Clean Code, Clean Architecture, and Code Complete 2.0 are your bibles for coding standards, patterns, and methodologies.

**Pull request closed without merging**